drift-client/src/main/java/io/airlift/drift/client/RetryPolicy.java

```diff
@@ -108,6 +110,10 @@ public ExceptionClassification classifyException(Throwable throwable, boolean id
             return new ExceptionClassification(Optional.of(TRUE), NORMAL);
         }
 
+        if (throwable instanceof TooLongMessageException) {
+            return new ExceptionClassification(Optional.of(FALSE), NORMAL);
```

Comment:
Use FALSE to be consistent with the other code (doesn’t matter except for consistency)